### ISSUE TITLE
Project acceptance criteria

### ISSUE THREAD

**Issue by carletes** [52North/ecmwf-dataset-crawl#19]
Just a bit of paperwork, @noerw, as required by the admin people here at ECMWF.

I suggest the following acceptance criteria to consider the project as finished:

* All code and documentation contained in a GitHub repo (this one), tagged as `v1.0` in the `master` branch.
* A top-level `Makefile` or shell script to build the web crawler.
* A top-level script to run the web crawler.
* Documentation on how to use the web crawler.

Basically what you have already sketched, as of today.

@noerw, @simonjirka, @matthesrieke: Does this look reasonable to you?

**Comment by matthesrieke** [comment]
@carletes : This looks perfectly fine to us. We will include this into our consolidation phase.

**Comment by carletes** [comment]
Thanks, @matthesrieke!
